src/components/ChatLog.js

```diff
@@ -0,0 +1,40 @@
+
+import React from 'react';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+const ChatLog = (props) => {
+    const entryComponents = props.entries.map((entry) => {
```

Comment:
The key prop should be for ChatEntry component, not on the div element. 

Also, instead of using the variable `entryComponents` to reference the return value from calling map() on props.entries, it's more common to directly iterate over the chat entries in your component like this without using a variable: 
```javascript
const ChatLog = (props) => {
    return (
        <div className="chat-log">
            {props.entries.map((entry) => (
                <ChatEntry 
                    id={entry.id}
                    sender={entry.sender}
                    body={entry.body}
                    timeStamp={entry.timeStamp}
                    liked={entry.liked}
                    key={entry.id} // this is where the key prop should be
                    onUpdateEntry={props.onUpdateEntry}
                />
            ))}
        <div/>
    )
};